client/src/components/tools.jsx

```diff
@@ -32,36 +33,36 @@ const Tools = () => {
     <div className="gm_tools">
       <div className="gm_tools-content">
         <Menu pointing>
-          <Menu.Item
-            disabled
-            active={selected === 'account'}
-            onClick={updateTab('account')}
-          >
-            User Management
-          </Menu.Item>
+          {' '}
           <Menu.Item
             active={selected === 'online'}
             onClick={updateTab('online')}
           >
             Who's Online
           </Menu.Item>
+          <Menu.Item
+            active={selected === 'account'}
+            onClick={updateTab('account')}
+          >
+            User Management
+          </Menu.Item>
```

Comment:
Why was this moved to after `Who's Online`?
If we want to reorder it, it should be done in a separate PR.

Reply:
Good eye on that one mate.